tendrl/commons/flows/import_cluster/__init__.py

```diff
@@ -217,99 +270,71 @@ def run(self):
-                                                                                                        req_min_ver,
-                                                                                                        req_rel)
+                    payload={"message": "Waiting for participant nodes %s to be "
+                                        "imported %s" % (node_list, integration_id)
                          }
                 )
             )
 
-            if int(maj_ver) < int(req_maj_ver) or \
-                int(min_ver) < int(req_min_ver):
-                Event(
-                    Message(
-                        job_id=self.parameters['job_id'],
-                        flow_id = self.parameters['flow_id'],
-                        priority="error",
-                        publisher=NS.publisher_id,
-                        payload={"message": "Error: Minimum required version (%s.%s.%s) doesnt match that of detected Gluster Storage (%s.%s.%s)" % (req_maj_ver,
-                                                                                                            req_min_ver,
-                                                                                                            req_rel,
-                                                                                                            maj_ver, min_ver, 0)
-                             }
-                    )
-                )
+            # An import is sucessfull once all Node[] register to /clusters/:integration_id/nodes/:node_id
+            while True:
+                _all_node_status = []
+                gevent.sleep(3)
+                for node_id in self.parameters['Node[]']:
+                    _status = NS.tendrl.objects.ClusterNodeContext(node_id=node_id).exists() and NS.tendrl.objects.ClusterTendrlContext(integration_id=integration_id).exists()
+                    _all_node_status.append(_status)
+                if _all_node_status:
+                    if all(_all_node_status):
+                        Event(
+                            Message(
+                                job_id=self.parameters['job_id'],
+                                flow_id = self.parameters['flow_id'],
+                                priority="info",
+                                publisher=NS.publisher_id,
+                                payload={"message": "Import Cluster completed for all nodes in cluster %s" % integration_id
+                                     }
+                            )
+                        )
 
-                raise FlowExecutionFailedError(
-                    "Detected gluster version: %s"
-                    " is lesser than required version: %s" %
-                    (
-                        detected_cluster.sds_pkg_version,
-                        reqd_gluster_ver
-                    )
+                        break
+    
+        except Exception as ex:
```

Comment:
this try except should contain all the code in this file (the entire of "run" function)